client/src/store.js

```diff
@@ -20,10 +20,31 @@ export default new Vuex.Store({
   },
   actions: {
     async fetchPipelines({ commit, state }) {
-      if (state.pipelines.length === 0) {
+      if (Object.keys(state.pipelines).length === 0) {
         const { data } = await getPipelineList();
-        commit('setPipelines', data);
-        return data;
+        // Sort list of pipelines in each category by name
+        Object.keys(data).forEach((key) => {
```

Comment:
You're only using `key` to get a value here.  Maybe cut out the middle-man and `Object.values`.